lib/generate-index.js

```diff
@@ -38,7 +38,19 @@ function generateIndex (playbook, pages) {
     // Exclude pages marked as "noindex"
     .filter(({ page, $ }) => {
       const $metaRobots = $('meta[name=robots]')
-      return !(($metaRobots && $metaRobots.attr('content') === 'noindex') || (page.asciidoc && page.asciidoc.attributes && page.asciidoc.attributes.noindex === ''))
+
+      const metaRobotNoIndex = $metaRobots && $metaRobots.attr('content') === 'noindex'
+      const pageNoIndex = page.asciidoc && page.asciidoc.attributes && page.asciidoc.attributes.noindex === ''
+      const noIndex = !(metaRobotNoIndex || pageNoIndex)
```

Comment:
If you want to negate the boolean value then the name should be `indexable`.
As an alternative, we could keep `noIndex` and negate the boolean value when we need to tell if the page should be indexed or not (ie. in the `return` statement below).

Reply:
Quite right. Sorry. I was confused. I'd prefer the no/yes not to be in the boolean name but for consistency with the way it is now I think the latestVersion should have the same naming convention. WDYT?

Reply:
I think it's fine 👍 
Indeed, consistency is the key and we can still revisit the naming convention later.